static/css/todo.css

```diff
@@ -147,7 +167,7 @@ input {
 
 /* Drag-and-drop feedback */
 .todo-item.drag-over {
-    border: 2px dashed var(--background-secondary-lm);
+    border: 125rem dashed var(--background-secondary-lm);
```

Comment:
125rem is really large for a border, should this be 0.125rem? That's the rough conversion from px to rem

Reply:
oh, this portion is not used currently since drag and drop only appends the item to the end of the list currently. I'm currently working on that function, but it might be more like an enhancement.

Reply:
Got it, sounds good!